# Starts and stops the sonar filter

# true to start | false to stop
bool enable
# filter initialization params
float32 x
float32 y
float32 stdev